src/DnsUtils.ts

```diff
@@ -0,0 +1,53 @@
+import DnsRecordsError, { DnsRecordsErrorCode } from "./errors/dnsRecordsError";
+import { CryptoRecords, DnsRecord, DnsRecordType } from "./publicTypes";
+import { isStringArray } from "./utils";
+
+export default class DnsUtils {
+
+  public toClassical(record: CryptoRecords): DnsRecord[] {
```

Comment:
I think we can have better naming. The word `classical` doesn't really suggest anything, and I can't see us following any standard here with this JSON schema for such naming to be justified.
I'd try to go with something like `flatten` (or `toList`) and `toKeyValue`, so that the name suggests the underlying response structure.